### DISCUSSION TITLE
在使用chatglm3与langchain报出CUDA error: device-side assert

### DISCUSSION THREAD

**Discussion by bearbee123** [zai-org/ChatGLM3#557] [Bad Case]
在使用_call函数中使用model.stream_chat或者model.chat方法如果接受到来自lanchain查询向量库RetrievalQA.from_chain_type返回的prompt后，进行多轮回答每次显存都会不断增加，直到显存爆掉出现CUDA error: device-side assert的错误
![29c38a52-3314-428e-8a22-6e23ed595512](https://github.com/THUDM/ChatGLM3/assets/79078255/627eddeb-4e94-43ff-adc7-b28ded09286d)
![339c9de7-3000-4e8a-a1cc-780157ccada7](https://github.com/THUDM/ChatGLM3/assets/79078255/7f72634e-7fdb-4f50-9366-23271e39f5e5)
已经不知道怎么办了，求助！！！

**Comment by zRzRzRzRzRzRzR** [comment]
#393 这里可以讨论
